docker/pytorch-aarch64/Dockerfile

```diff
@@ -292,10 +292,11 @@ COPY patches/onednn.patch $PACKAGE_DIR/.
 RUN $PACKAGE_DIR/build-pytorch.sh
 # Install torchvision, torchdata and torchtext
 RUN pip install --no-cache-dir torchvision==$TORCHVISION_VERSION \
-    torchdata==$TORCHDATA_VERSION
+    torchtext==$TORCHTEXT_VERSION
+
 ENV VENV_PACKAGE_DIR=$VENV_DIR/lib/python$PY_VERSION/site-packages
-COPY scripts/build-torchtext.sh $PACKAGE_DIR/.
```

Comment:
Have you run all the examples on your finished build? (see https://github.com/ARM-software/Tool-Solutions/tree/master/docker/pytorch-aarch64/examples). Do they run without issue?

We build torchtext from source as we had encounter problems (un resolved symbols, as I recall) running the Torchtext examples. Could you confirm there are no issues with the examples? If you see failures in the torchtext example (https://github.com/ARM-software/Tool-Solutions/tree/master/docker/pytorch-aarch64/examples#torchtext-article-reading) then I think the from-source build of torchtext will need to be reinstated.

Reply:
Thanks @nSircombe ! Looks like the symbol mismatch error still exists for torchtext, so I left the source build option for now. btw, the newer version, v0.13.0 needs cmake > =3.18, hence I added cmake build script additionally. And I've also executed to all the examples from the container and they were all passing now.

Reply:
> the newer version, v0.13.0 needs cmake > =3.18,

That's slightly annoying! 

> I added cmake build script 

I think it would be better to stick with the precedence of getting tools like cmake in the base image rather than adding a new version this late on. 
But I think we can avoid a build-from-source as Kitware maintain an apt-repository that should let us continue to `apt-get install` cmake into the base image, we just have to add the new repo.

Something like this change to the Dockerfile should do it:


```
diff --git a/docker/pytorch-aarch64/Dockerfile b/docker/pytorch-aarch64/Dockerfile
index ef624c3..9b1d2c9 100644
--- a/docker/pytorch-aarch64/Dockerfile
+++ b/docker/pytorch-aarch64/Dockerfile
@@ -27,11 +27,22 @@ ENV PY_VERSION="${default_py_version}"

 RUN if ! [ "$(arch)" = "aarch64" ] ; then exit 1; fi

-#Install core OS packages
+
 RUN apt-get -y update && \
-    apt-get -y install software-properties-common && \
-    add-apt-repository ppa:ubuntu-toolchain-r/test && \
     apt-get -y install \
+      software-properties-common \
+      wget
+
+# Add additional repositories
+RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null \
+      | gpg --dearmor - \
+      | tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null
+
+RUN add-apt-repository ppa:ubuntu-toolchain-r/test && \
+    add-apt-repository 'deb https://apt.kitware.com/ubuntu/ focal main'
+
+# Install core OS packages
+RUN apt-get -y install \
       accountsservice \
       apport \
       at \
@@ -96,7 +107,6 @@ RUN apt-get -y update && \
       ufw \
       uuid-runtime \
       vim \
-      wget \
       xz-utils \
       zip \
       zlib1g-dev
diff --git a/docker/tensorflow-aarch64/scripts/build-tensorflow.sh b/docker/tensorflow-aarch64/scripts/build-tensorflow.sh
index 96de756..0e4f658 100755
--- a/docker/tensorflow-aarch64/scripts/build-tensorflow.sh
+++ b/docker/tensorflow-aarch64/scripts/build-tensorflow.sh
@@ -72,7 +72,7 @@ if [[ $ONEDNN_BUILD ]]; then
         config_flags="$config_flags --config=mkl_aarch64_threadpool"
     else
         config_flags="$config_flags --config=mkl_aarch64"
-        compile_flags="$compile_flags --copt=-fopenmp"
+        #compile_flags="$compile_flags --copt=-fopenmp"
     fi
     if [[ $ONEDNN_BUILD == 'reference' ]]; then
         tf_backend_desc="oneDNN - reference."
```

Reply:
great! thanks, @nSircombe ! I will replace my cmake build script with this. thanks.